app/src/main/java/org/jellyfin/mobile/MainActivity.kt

```diff
@@ -103,7 +113,11 @@ class MainActivity : AppCompatActivity() {
         requestCode: Int,
         permissions: Array<out String>,
         grantResults: IntArray
-    ) = permissionRequestHelper.handleRequestPermissionsResult(requestCode, permissions, grantResults)
+    ) {
+        super.onRequestPermissionsResult(requestCode, permissions, grantResults)
+
+        permissionRequestHelper.handleRequestPermissionsResult(requestCode, permissions, grantResults)
+    }
```

Comment:
Ah lol, I wanted to fix this *for ages*

Reply:
We should actually move it to contracts but not in the scope of this PR